app/menus/menu.js

```diff
@@ -12,7 +12,6 @@ const darwinMenu = require('./menus/darwin');
 module.exports = (createWindow, updatePlugins) => {
   const commands = getKeymaps().commands;
   const menu = [
-    (process.platform === 'darwin' ? darwinMenu(commands) : []),
```

Comment:
I tried `(process.platform === 'darwin' ? [darwinMenu(commands)] : []),` and it still causes a blank menu entry on Windows